torch/csrc/jit/tensorexpr/ir.h

```diff
@@ -797,6 +797,14 @@ class Intrinsics : public CallNode<Intrinsics> {
   IntrinsicsOp op_type_;
 };
 
+class NoOp : public ExprNode<NoOp> {
```

Comment:
Minot nitpicking: unless you see this NoOp used in many more places, I'd rather give it a name that is clearly internal. For example: PlaceHolderExpr? Or put into an implementation namespace, such as: impl::PlacerHolder?